src/AuditManager.php

```diff
@@ -60,7 +60,7 @@ public function getConfiguration()
     /**
      * NEXT_MAJOR: Use `\Doctrine\ORM\EntityManagerInterface` for argument 1.
      *
-     * @return AuditReader
+     * @return AuditReader<object>
      */
     public function createAuditReader(EntityManager $em)
     {
```

Comment:
I guess this means that when calling `AuditManager::createAuditReader()` would be always of type `object` and calling then for example `AuditReader::find(Foo::class)` would always return and object of type `object` instead of a `Foo` object

Reply:
https://github.com/sonata-project/EntityAuditBundle/pull/488#issuecomment-1107778889 might be a good way to solve this.